src/app/components_ngrx/work-item-cell/work-item-cell.component.html

```diff
@@ -42,7 +42,7 @@
 <div *ngIf="col === 'state'">
   <span class="padding-right-5 padding-top-4" almIcon [iconType]="row.state">
   </span>
-  <span>
+  <span style="white-space: normal;">
```

Comment:
Why did you add this inline style?

Reply:
considering it as temporary solution

Reply:
Line height for this should be also normal otherwise it looks like this

![screenshot from 2018-08-24 13-48-50](https://user-images.githubusercontent.com/2561818/44573536-899c5800-a7a4-11e8-9396-f26f78ef69bc.png)